lms/djangoapps/certificates/tests/test_api.py

```diff
@@ -401,6 +408,41 @@ def test_no_certificates_for_user(self):
             []
         )
 
+    @patch.dict(settings.FEATURES, {'CERTIFICATES_HTML_VIEW': True})
+    def test_get_web_certificate_url(self):
+        """
+        Test the get_certificate_url with a web cert course
+        """
+        expected_url = reverse(
+            'certificates:render_cert_by_uuid',
+            kwargs=dict(certificate_uuid=self.uuid)
+        )
+        cert_url = certs_api.get_certificate_url(course_id=self.course_1.id, uuid=self.uuid)
+        self.assertEqual(expected_url, cert_url)
+
+        expected_url = reverse(
+            'certificates:html_view',
+            kwargs={
+                "user_id": str(self.student.id),
+                "course_id": unicode(self.course_1.id),
+            }
+        )
+
+        cert_url = certs_api.get_certificate_url(user_id=self.student.id, course_id=self.course_1.id)
+        self.assertEqual(expected_url, cert_url)
+
+    @patch.dict(settings.FEATURES, {'CERTIFICATES_HTML_VIEW': True})
+    def test_get_pdf_certificate_url(self):
+        """
+        Test the get_certificate_url with a pdf cert course
+        """
+        cert_url = certs_api.get_certificate_url(
+            user_id=self.student.id,
+            course_id=self.course_2.id,
```

Comment:
These names are not readable at all for someone approaching the code for the first time. We'd all be better served if you took this opportunity to rename them `web_cert_course` and `pdf_cert_course`.